config.lua

```diff
@@ -24,6 +24,7 @@ maxPlayers = 0
 motd = "Welcome to The Forgotten Server!"
 onePlayerOnlinePerAccount = true
 allowClones = false
+allowWalkthrough = true
```

Comment:
Nope, in Tibia players are walkthroughable by default. This is not a TFS-only feature, maybe you didn't know it has been implemented in Tibia :P